src/mscorlib/src/System/Globalization/CultureData.cs

```diff
@@ -626,14 +626,19 @@ internal static CultureData GetCultureData(String cultureName, bool useUserOverr
             return culture;
         }
 
-        private static unsafe string NormalizeCultureName(string name, out bool isNeutralName)
+        private static string NormalizeCultureName(string name, out bool isNeutralName)
         {
             isNeutralName = true;
             int i = 0;
 
-            Debug.Assert(name.Length <= LOCALE_NAME_MAX_LENGTH);
+            if (name.Length > LOCALE_NAME_MAX_LENGTH)
+            {
+                // Theoretically we shouldn't hit this exception.
+                throw new ArgumentException(SR.Format(SR.Argument_InvalidId, nameof(name)));
+            }
+
+            Span<char> pName = stackalloc char[name.Length];
```

Comment:
It is not a pointer anymore. Could you please rename this to `neutralName` ?

Reply:
will do.